scripts/docker-bcoin-init.js

```diff
@@ -4,54 +4,53 @@ const bcoin = require('bcoin');
 const fs = require('fs');
 const path = require('path');
 const os = require('os');
+const blgr = require('blgr');
 
-let prefix = process.env.prefix
-  ? process.env.prefix
-  : `${os.homedir()}/.bcoin/`;
-const initScript = process.env.BCOIN_INIT_SCRIPT;
-
-prefix = prefix.replace('~', os.homedir());
-// create the node with our custom configs
-const node = new bcoin.FullNode({ env: true });
-
-// checking if walletdb dir already exists (i.e. was mounted as volume)
-// need to do before using plugin because that will create the dir
-// if not already there
-const fileList = fs.existsSync(prefix) ? fs.readdirSync(prefix) : [];
-const hadWalletDB = fileList.indexOf('wallet') > -1;
-
-if (!node.config.bool('no-wallet') && !node.has('walletdb')) {
-  const walletPlugin = bcoin.wallet.plugin;
-  node.use(walletPlugin);
-}
-
-node.on('error', e => console.error('There was an error: ', e));
+// global variables
+let logger;
+let node;
 
 (async () => {
-  await node.ensure();
-  await node.open();
-  await node.connect();
-  node.startSync();
-})()
-  .then(async () => {
-    console.log('bcoin node is started');
-    // want to set a coinbase address in case of mining
-    const wdb = node.require('walletdb').wdb;
-    const primary = wdb.primary;
-    const coinbase1 = await primary.receiveAddress();
-    const coinbase2 = await primary.receiveAddress();
-    await node.miner.addAddress(coinbase1);
-    await node.miner.addAddress(coinbase2);
+  try {
+    logger = new blgr({
+      level: 'debug'
```

Comment:
TODO: don't hardcode level

Reply:
Good call. Seems like a good thing to get from the env variables.